### ISSUE TITLE
Accessibility issues on the Welcome page

### ISSUE THREAD

**Issue by shrilakshmishastry** [usebruno/bruno#1907]
List of accessibility issues on the welcome page:

1. bruno icon shouldn’t be accessible as it’s decorative
2. The wrong HTML is used for the button, line numbers 62 ,67, 71 in the Welcome component. All of these need to be buttons or link
3. in all links SVG should be aria-hidden

List of accessibility issues on sidebar footer icons

1. wrong HTML used
2. wrong roles used

List of accessibility issues in Modal
1. When the modal is open windows under a modal dialog are accessible by the keyboard

**Comment by bpoulaindev** [comment]
I initially had a PR aiming to replace the icons, use buttons, improve accessibility with keyboard navigation... : #1359 this is not really planned for the moment, too many core functionalities to work on. But feel free to send a PR to fix what you mentioned

**Comment by shrilakshmishastry** [comment]
> I initially had a PR aiming to replace the icons, use buttons, improve accessibility with keyboard navigation... : #1359 this is not really planned for the moment, too many core functionalities to work on. But feel free to send a PR to fix what you mentioned

fixed here https://github.com/usebruno/bruno/pull/1906
